proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpTraceUtils.java

```diff
@@ -86,6 +86,24 @@ public class OtlpTraceUtils {
         }
       };
 
+  public static String getAttrValByKey(List<KeyValue> attributesList, String key) {
+    Optional<KeyValue> sourceAttr =
+        attributesList.stream().filter(kv -> key.equals(kv.getKey())).findFirst();
```

Comment:
Can this be replaced with a call to `getAttrByKey()`?

Reply:
Updated the code